app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post, counter_cache: :comments_counter
+end
```

Comment:
It is required for the comment model to have a method that updates the comments counter for a post. 😸

Reply:
Hello @alexander16108, thank for the review. I am actually using the **[counter_cache](https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-counter-cache)** **method** provided by the belongs_to association reference to update the comments counter. Does it not count as a method?